.github/workflows/generate.yml

```diff
@@ -0,0 +1,55 @@
+name: Generate
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_level:
+        description: "Bump version"
+        required: true
+        default: "patch"
+        type: choice
+        options:
+        - major
+        - minor
+        - patch
+
+jobs:
+  Generate:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true
+        ruby-version: 3.1
+    - name: Bump version
+      id: bump_version
+      run: echo "::set-output name=version::$(ruby .github/version.rb ${{ github.event.inputs.version_level }})"
+    - name: Clean repo
+      run: ruby .github/clean.rb
+    - uses: actions/setup-node@v2
+    - run: |
+        npm install @openapitools/openapi-generator-cli -g
+        openapi-generator-cli version
+    - run: |
+        openapi-generator-cli generate \
+          -i https://raw.githubusercontent.com/mxenabled/openapi/master/openapi/mx_platform_api_beta.yml \
+          -g ruby \
+          -c ./openapi/config.yml \
+          -t ./openapi/templates
+    - name: Create branch
+      run: git checkout -b "openapi-generator-${{ steps.bump_version.outputs.version }}"
+    - name: Create commit
+      run: |
+        git config user.name "openapi_generator"
+        git config user.email "openapi_generator@mx.com"
```

Comment:
What do you think about using our team email here?

Reply:
Good idea. Just updated.